### DISCUSSION TITLE
source map not generated in dev build with chakra ui

### DISCUSSION THREAD

**Discussion by stardvst** [chakra-ui/chakra-ui#8581] [General]
### Summary

i've had my dev build running without errors, and now i get error after adding chakra ui to my app:
`GET /_next/static/chunks/app/dashboard/index.mjs.map 404 in 1443ms`

i haven't changed next.config.mjs:
```
/** @type {import('next').NextConfig} */
const nextConfig = {};
export default nextConfig;
```

i can see that before adding chakra ui, map files weren't generated, but maybe chakra needs and requests them, and can't find them.
what could be the reason?

**Comment by segunadebayo** [comment]
This is Next not emitting source maps, not a Chakra bug. Check your Next or webpack sourcemap config.
